### PR TITLE
fix(root): Fix eslint issues

### PR DESCRIPTION
**Pull request by yasincaliskan**
### Description
- Some eslint rule violations were ignored.
- `"react/jsx-handler-names"` rule was added to `.eslintrc`.
- `.eslintrc` was added to `.eslintignore`.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -28,8 +28,7 @@ export default [
       FileUploadButton: "src/button/file-upload/FileUploadButton.tsx",
       Spinner: "src/spinner/Spinner.tsx",
       Countdown: "src/countdown/Countdown.tsx",
-      ProgressBar: "src/progress-bar/ProgressBar.tsx",
-      Textarea: "src/form/textarea/Textarea.tsx"
```
Comment (jamcry) [inline rollup.config.js:32]: Was this a mistake? Why are we removing `Textarea` ?

**Review by bilgeyucel — APPROVED** [review]
Looks good! Apart from these updates, let's open an issue to update eslint package versions

File: rollup.config.js (hunk 1 above)
Comment (yasincaliskan) [reply rollup.config.js]: Ops! I supposed that there is duplicated `Textarea` but there wasn't. My bad, sorry! 😅 

Actually, we can provide finding eslint issues while npm is running. Should we add a script to `package.json` for this?

Comment (jamcry) [reply rollup.config.js]: Yes, I think we should add a step to build command for linting. This way, we can make sure there are no errors before publishing to npm.
I think we can create an issue (or update, if you opened one for the issue Bilge mentioned below), and add it later.

**Review by jamcry — APPROVED** [review]

**Review by edizcelik — APPROVED** [review]

**Review by mucahit — APPROVED** [review]

**Pull request merged**